src/theme/components/button.ts

```diff
@@ -3,19 +3,82 @@ import { defineStyleConfig } from "@chakra-ui/react";
 export const buttonTheme = defineStyleConfig({
   baseStyle: {
     backgroundColor: "primary.600",
+    borderRadius: "base",
+    // token?
     color: "white",
     px: 6,
     py: 3,
   },
   sizes: {
-    sm: {},
-    md: {},
-    lg: {},
+    sm: {
+      height: 11,
```

Comment:
We should stay away from hard coding the height of the button so that they can accommodate wrapped text. It looks like the font sizes are actually different in the Figma specs for each size. If you replace these `height`s with `fontSize`s pointing to font size tokens, the total height of the button should work out to what you see in Figma.

I know it's not clear just from the designs whether we wanted hard coded or "auto" height here. I'll mention that to Design as an opportunity to improve hand off.